code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -909,16 +915,33 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 			update_icons()
 		return
 
-/mob/living/silicon/robot/verb/unlock_own_cover()
+/mob/living/silicon/robot/verb/toggle_own_cover()
 	set category = "Robot Commands"
-	set name = "Unlock Cover"
-	set desc = "Unlocks your own cover if it is locked. You can not lock it again. A human will have to lock it for you."
-	if(locked)
-		switch(alert("You can not lock your cover again, are you sure?\n      (You can still ask for a human to lock it)", "Unlock Own Cover", "Yes", "No"))
-			if("Yes")
-				locked = 0
-				update_icons()
-				to_chat(usr, "You unlock your cover.")
+	set name = "Toggle Cover"
+	set desc = "Toggles the lock on your cover."
+	var/is_ert = has_special_laws()
+	//Standard borgs
+	if(!is_ert)
+		if(locked)
+			switch(alert("You can not lock your cover again, are you sure?\n      (You can still ask for a human to lock it)", "Unlock Own Cover", "Yes", "No"))
+				if("Yes")
+					locked = 0
+					update_icons()
+					to_chat(usr, "You unlock your cover.")
+	//Special borgs
+	else
+		if(locked)
+			switch(alert("Are you sure?", "Unlock Own Cover", "Yes", "No"))
+				if("Yes")
+					locked = 0
+					update_icons()
+					to_chat(usr, "You unlock your cover.")
+		else
+			switch(alert("Are you sure?", "Lock Own Cover", "Yes", "No"))
+				if("Yes")
+					locked = 1
+					update_icons()
+					to_chat(usr, "You lock your cover.")
```

Comment:
```suggestion
	if(is_ert)
		if(alert("Are you sure?", locked ? "Unlock Cover" : "Lock Cover", "Yes", "No") == "Yes")
			locked = !locked
			update_icons()
			to_chat(usr, "<span class='notice'>You [locked ? "lock" : "unlock"] your cover.</span>")
    return
	if(locked)
    to_chat(usr, "You cannot lock your cover yourself. Find a robotocist.")
    return
	if(alert("You cannnot lock your own cover again. Are you sure? You will need a robotocist to re-lock you.", "Unlock Own Cover", "Yes", "No") == "Yes")
		locked = !locked
		update_icons()
		to_chat(usr, "<span class='notice'>You unlock your cover.</span>")
```

Reply:
`if(alert("Are you sure?", >>>locked ? "Unlock Cover" : "Lock Cover"<<<, "Yes", "No") == "Yes")`
Didn't know you could do that outside of text!

`locked = !locked`
Didn't know you could do this at all!